message-index/messages/GHC-84077/example-parsing/before/index.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
I think this file should be in the `example/parsing/` folder. It currently looks like this for me:

![image](https://user-images.githubusercontent.com/7516482/173193910-7f104993-bd4f-4d23-ab4f-72f828329539.png)

Reply:
Moved to `example/parsing `directory